### PR TITLE
Added an XSD schema for this bundle

### PR DESCRIPTION
**Pull request by WouterJ**

### CODE REVIEW

**Comment by dbu** [comment]
can you maybe add a test that loads a configuration example and checks if the schema works?

File: Resources/config/schema/menu-1.0.xsd [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+            <xsd:element name="twig" type="twig" minOccurs="0"/>
+        </xsd:all>
+
+        <xsd:attribute name="templating" type="xsd:boolean"/>
+        <xsd:attribute name="default-renderer" type="xsd:string"/>
+    </xsd:complexType>
+
+    <xsd:complexType name="providers">
+        <xsd:attribute name="builder-alias" type="xsd:boolean"/>
+        <xsd:attribute name="container-aware" type="xsd:boolean"/>
```
Comment (stof) [inline Resources/config/schema/menu-1.0.xsd]: this needs to be updated to account for #273

**Comment by stof** [comment]
@WouterJ ping

**Comment by dbu** [comment]
lets add some configuration unit tests as well to make sure it does not break in the future.

**Comment by WouterJ** [comment]
Updated the XSD and added some tests.

**Comment by dbu** [comment]
:+1:

**Pull request merged**